packages/zosfiles/src/api/methods/hDelete/doc/IDeleteOptions.ts

```diff
@@ -0,0 +1,24 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { IMigrateOptions } from "../../hMigrate";
+
+/**
+ * This interface defines the options that can be sent into the delete data set function.
+ */
+export interface IDeleteOptions extends IMigrateOptions {
```

Comment:
Ive extended the previous interface `IMigrateOptions`, I feel there could be a better implementation, just not sure what so I am open to suggestions

Reply:
I personally don't dislike reusing interfaces whenever possible. 😋 

how does `IDatasetUtils` sound? and the only required field is the `request` property? (and wait and purge are optional) 😋

Reply:
Hey @zFernand0, thanks for taking the time to review! Wasn't sure on if reusing interfaces was a good implementation, just needed to make sure. For the idea of `IZosFilesUtils`, just to make sure we're on the same page, it would just be a base "options" handler? so all the `IMigrateOptions`,`IRecallOptions`,`IDeleteOptions` "options" would all be inside this `IZosFilesUtils`(meaning API's would use this to append the options to the `payload`). As for the `request` property, could you explain which property you are referring to?